e2e/playwright/testing-segment-overlays.spec.ts

```diff
@@ -1628,7 +1709,7 @@ profile003 = startProfileAt([64.39, 35.16], sketch001)
 
     // Verify overlays are visible
     // 3 for the three point arc, and 4 for the 3 segments (arc has two)
-    await expect(page.getByTestId('segment-overlay')).toHaveCount(7)
+    await expect(page.getByTestId('segment-overlay')).toHaveCount(8)
```

Comment:
there's going to be a lot more tests that need integers updated like this, because I've added a new overall into nearly ever sketch because it's for `startProfile`.

At the time of writing I've only fixed a handle, but will fix the rest when I get up in the morning

Reply:
Ok yeah there's bunch more to fix, I think we can help!